src/otx/core/model/detection.py

```diff
@@ -190,9 +195,17 @@ def __init__(
 
     def forward_explain(
         self,
-        inputs: DetBatchDataEntity,
+        inputs: DetBatchDataEntity | TileBatchDetDataEntity,
     ) -> DetBatchPredEntity:
         """Model forward function."""
+        from otx.algo.explain.explain_algo import get_feature_vector
+
+        if isinstance(inputs, OTXTileBatchDataEntity):
+            return self.forward_tiles(inputs)
+
+        self.model.feature_vector_fn = get_feature_vector
+        self.model.explain_fn = self.get_explain_fn()
```

Comment:
Just a comment, not to be done in this PR.

`get_feature_vector` is a function itself
`get_explain_fn()` returns the function
But the naming convention is the same, which is confusing.

What about rename `get_feature_vector` -> `feature_vector_fn`?

Reply:
Will do in the future PR, thanks for pointing out